ui/src/pages/ViewEvent.jsx

```diff
@@ -52,6 +53,8 @@ class ViewEvent extends React.Component {
   deleteEvent = () => {
     const newTrip = Trip.fromObject(this.props.trip.toObject());
     newTrip.events = newTrip.events.filter(event => event.id !== this.props.event.id);
+    if (newTrip.background === ImageModel.getBlackImage()) 
+      newTrip.background = event.images[0];
```

Comment:
Which `event` is this referring to?

Reply:
I think this isn't necessary to add because we're going to get rid of all of the random `if` checks everywhere and just implement a `background()` function in the `Trip` model.